templates/taxi/manufacturer_delete_confirmation.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{%  block content %}
+  <h2>Delete this manufacturer</h2>
+  <p>
+    Are you sure you really want to delete the information of this manufacturer?
+    ({{ manufacturer.name }})
+    <br>
+    <i>All cars will be lost then.</i>
+  </p>
+  <form action="" method="post">
+  {%  csrf_token %}
+  <input class="btn btn-danger" type="submit" value="Yes, delete">

```

Comment:
it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something